tests/contrib/test_torch.py

```diff
@@ -174,6 +177,25 @@ def test_torch_import(tmp_path, net, torchsave):
     assert isinstance(meta.model_type, TorchModel)
 
 
+def test_torch_import_in_separate_shell():
+    tmp_path = mkdtemp()
+    path = make_posix(os.path.join(tmp_path, "model"))
+    m = torch.nn.Linear(5, 1)
```

Comment:
does this break without cloudpickle?

Reply:
what breaks? declaration of `m = torch.nn.Linear(5, 1)`? Nope.
only loading from a separate terminal breaks as described in the issue here: https://github.com/iterative/mlem/issues/415

Reply:
Yes, I mean the entire test. Original problem was with custom net class which you do not use in this test, so I'm asking if you testing the same case

Reply:
Aah yes, you are right, this will pass in both cases, I will change to a custom net here.

Reply:
done